firmware/thruna/FSM/Test/Src/Test_StateMachine.cpp

```diff
@@ -111,11 +111,7 @@ class FsmStateMachineTest : public BaseStateMachineTest
         TearDownObject(wheels, App_Wheels_Destroy);
     }
 
-    void UpdateClock(struct StateMachine *state_machine, uint32_t current_time_ms) override
-    {
-        UNUSED(state_machine);
-        App_Timer_SetCurrentTimeMS(current_time_ms);
-    }
+    void UpdateClock(struct StateMachine *state_machine, uint32_t current_time_ms) override { UNUSED(state_machine); }
```

Comment:
Not sure what happened to the formatting on this one, please space it out, running the python script `scripts/clang_format/fix_formatting.py` might handle it for you